zdict/dictionaries/moe.py

```diff
@@ -96,3 +97,102 @@ def query(self, word: str):
                  )
 
         return record
+
+
+def is_other_format(char):
+    return unicodedata.category(char) != 'Cf'
+
+
+def remove_cf(data):
+    return ''.join(filter(is_other_format, data))
+
+
+def clean(data, clean_cf=False):
+    '''
+    Clean the word segmentation
+
+    remove "`~" and things in Unicode 'Cf' category
+    '''
+    data = data.translate(str.maketrans('', '', '`~'))
+    if clean_cf:
+        return remove_cf(data)
+    else:
+        return data
```

Comment:
What is the reason for exposing those helper functions (`is_other_format`, `remove_cf` and `clean`) globally instead of making them as private funcs of `MoeDictTaiwanese`?